core/src/main/scala/cats/MonadFilter.scala

```diff
@@ -4,15 +4,15 @@ import simulacrum.typeclass
 
 /**
  * a Monad equipped with an additional method which allows us to
- * create an "Empty" value for the Monad (for whatever "empty" makes
+ * create an "empty" value for the Monad (for whatever "empty" makes
  * sense for that particular monad). This is of particular interest to
  * us since it allows us to add a `filter` method to a Monad, which is
  * used when pattern matching or using guards in for comprehensions.
  */
-@typeclass trait MonadFilter[F[_]] extends Monad[F] {
+@typeclass trait MonadFilter[F[_]] extends Monad[F] with FunctorFilter[F] {
 
   def empty[A]: F[A]
```

Comment:
It seems like a FunctorFilter had to have an empty value too. We could just filter everything from any value and get there, no?

Seems like empty might almost be there.